test/graphql-api/multipart.mocks.ts

```diff
@@ -0,0 +1,38 @@
+import { setupWorker, graphql } from 'msw'
+
+const worker = setupWorker(
+  graphql.mutation<
+    {
+      multipart: {
+        file1?: string
+        file2?: string
+        files?: string[]
+        plainText?: string
+      }
+    },
+    {
+      file1?: File
+      file2?: File
+      files?: File[]
+      plainText?: string
+    }
+  >('UploadFile', async (req, res, ctx) => {
+    const { file1, file2, files, plainText } = req.variables
+    const filesResponse: string[] = []
```

Comment:
What do you think if we use `Promise.all` and a plain `files.map` instead of the `for`/`of` loop?

```js
const { files = [] } = req.variables
const filesResponse = await Promise.all(files.map(file => file.text())
```

It's going to be less code, making this example more focused. I wonder to hear your opinion.